### ISSUE TITLE
Make "Submit" button accessible

### ISSUE THREAD

**Issue by aSquare14** [ifmeorg/ifme#957]
# Description 
- Presently, user cannot submit a group or form by entering the "Enter" key, he/she has to click on "Submit"
- Make this component accessible.

**Comment by julianguyen** [comment]
Great call out, I actually worked on this in new redesign implementation https://github.com/ifmeorg/ifme/pull/942

If you wanna check it out, check out the static-pages-redesign in this repo and open Storybook :)
